resources/extruders/zonestar_printer_mixvtool_0.def.json

```diff
@@ -0,0 +1,17 @@
+{
+    "version": 2,
+    "name": "V TOOL 0",
+    "inherits": "fdmextruder",
+    "metadata": {
+        "machine": "ZONESTAR M3",
```

Comment:
That ID doesn't exist.

Reply:
Sorry, I don't quite understand what you mean "that ID"?

Reply:
This is referring to a printer with the file name `ZONESTAR M3.def.json`. This file doesn't exist, causing the automated tests to reject this PR. Your printer definition was called `zonestar_m3_printer`.